authors/apps/authentication/views.py

```diff
@@ -23,6 +35,45 @@ def post(self, request):
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
         serializer.save()
+
+        # get the user by their username
+        user_to_activate = User.objects.get(username=user['username'])
+
+        # Get the email of the user
+        to_email = user_to_activate.email
+
+        # Encode the user id
+        user_pk = user_to_activate.pk
+        user_id = urlsafe_base64_encode(force_bytes(user_pk)).decode()
+
+        # Generate the token
+        token = account_activation_token.make_token(user_to_activate)
+
+        # pass data to the reverse url
+        url = "{}".format(reverse('authentication:activate', kwargs={
+            'pk': user_id,
+            'token': token}
```

Comment:
I believe the curly brace should be in a new line. Refer to the [indentation](https://www.python.org/dev/peps/pep-0008/#indentation) sub-section of the Code layout section of the PEP8 standards.